src/app/user/login/components/login/login.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnInit } from '@angular/core';
+import {FormBuilder, FormControl} from "@angular/forms";
+import {User} from "../../models/user-model";
+import {LoginService} from "../../login.service";
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+export class LoginComponent implements OnInit {
+
+  constructor(
+    private formBuilder: FormBuilder,
+    private loginService: LoginService
+              ) { }
+
+  ngOnInit(): void {
+  }
+
+  formValues = this.formBuilder.group({
+    username:new FormControl(''),
+    password:new FormControl(''),
+  })
+
+  onSubmit() {
+    //let user : User = new User(this.formValues.value.username, bcrypt.hashSync(this.formValues.value.password, 10));
```

Comment:
maybe we can delete this commented code before merging, not visible password (hashing) will be managed in another ticket
```suggestion
    //let user : User = new User(this.formValues.value.username, bcrypt.hashSync(this.formValues.value.password, 10));
```